operations/README.md

```diff
@@ -151,11 +151,14 @@ This section describes all operations supported by the API, organised here by th
 | [Update routing rules](routingrules.md#update-routing-rules) | Updates selected routing rules |
 | [Delete routing rules](routingrules.md#delete-routing-rules) | Deletes selected routing rules |
 
-## Service orders
+## Service order notes
```

Comment:
```suggestion
## Service orders
```
Actually, this one didn't need to be changed. The individual page with the operations on it is called `Service order notes` but the _theme_ under which it's listed on the Operations home page can be just `Service orders`. So if we add any new operations that have anything remotely to do with service orders we would group them together here under the same _theme_. Hope that makes sense.